added appointmentChangedCallback

daviddbal opened the pull request:
currently handles drag-and-drop and expand time changes but could be
added to any changes to an appointment by Agenda

daviddbal committed:
added appointmentChangedCallback

currently handles drag-and-drop and expand time changes but could be
added to any changes to an appointment by Agenda

tbee commented on jfxtras-agenda/src/main/java/jfxtras/scene/control/agenda/Agenda.java:
```diff
@@ -348,7 +347,19 @@ private void constructAppointmentGroups() {
+     * If so, you need to set this callback method and open your own window.
+     */
+    public ObjectProperty<Callback<Appointment, Void>> appointmentChangedCallbackProperty() { return appointmentChangedCallbackObjectProperty; }
+    final private ObjectProperty<Callback<Appointment, Void>> appointmentChangedCallbackObjectProperty = new SimpleObjectProperty<Callback<Appointment, Void>>(this, "editAppointmentCallback", null);
```
The string is incorrect, "appointmentChangedCallback"

daviddbal committed:
fixed string error in appointmentChangedCallback

daviddbal committed:
fixed string error in appointmentChangedCallback

daviddbal committed:
fixed unwanted changes caused by browser editor

Conflicts:
	jfxtras-
agenda/src/main/java/jfxtras/scene/control/agenda/Agenda.java

daviddbal commented:
I edited the string in the browser editor and it changed the entire file.  I think it is just some formatting changes, but it affected the entire Agenda class.  I won't use the browser editor again.

I went to the original in Eclipse and fixed it, but the history now is messed up.  If you don't want the unnecessary commits in the history I can fix it later. I'm too tired to do it now.
